mkdocs/plugins.py

```diff
@@ -5,10 +5,10 @@
 from __future__ import annotations
 
 import logging
+import sys
 from collections import OrderedDict
 from typing import Any, Callable, Dict, Optional, Sequence, Tuple, TypeVar, overload
 
-import importlib_metadata
```

Comment:
I just had a nitpick that I like to still keep the order of import statements - this is kinda between stdlib and third party, so: https://github.com/oprypin/mkdocs/commit/55da0f892a0eeff731b849486f751c0596ca0506

Reply:
Sure, I put it below because that's usually how isort likes it. Suggestion applied :slightly_smiling_face: